### PR TITLE
Add support for Vec::operator(<, >=, >)

### PR DESCRIPTION
**Pull request by BenjaminW3**

### CODE REVIEW

**Comment by sbastrakov** [comment]
I totally understand the convenience of having it, having done similar stuff myself. Just to point out, from some perspective it is not logical. Having just element-wise >= and <= is fine, each defines a partial order on the set of vectors. However, with element-wise > present, now >= does not mean "greater or equal" in the sense of that element-wise greater and natural equality, and it actually can not be easily expressed as a combination of that two. Again, I still think the convenience overweights this little issue.

**Comment by BenjaminW3** [comment]
Ready for review!

File: include/alpaka/vec/Vec.hpp [hunk 1]
```diff
@@ -721,7 +764,7 @@ namespace alpaka
         }
 
         //-----------------------------------------------------------------------------
-        //! \return The element wise less than relation of two vectors.
+        //! \return The element wise less equal than relation of two vectors.
```
Comment (sbastrakov) [inline include/alpaka/vec/Vec.hpp]: I think 'element wise' present in many places is not a correct form, should be element-wise or elementwise https://en.wiktionary.org/wiki/elementwise

Also here 'less equal than' should probably be phrased as 'less than or equal'

Comment (BenjaminW3) [reply include/alpaka/vec/Vec.hpp]: Fixed

**Pull request merged**